src/router/ProtectedRouter.tsx

```diff
@@ -1,11 +1,24 @@
-import { Outlet } from "react-router-dom";
+import { Navigate, Outlet } from "react-router-dom";
 import LandingPage from "../pages/LandingPage/LandingPage";
 import { useAuthContext } from "../store/auth/AuthContext";
+import { userRoles } from "../constants/userRoles";
+import { routes } from "../constants/routes";
 
-const ProtectedRouter = () => {
-  const { isLoggedIn } = useAuthContext();
+type Props = {
+  role?: string;
+};
```

Comment:
I think we need to give more context to this prop name, when using it from parent component we don't know from name `role` what it will do
Think of some name that will show what is the purpose of that prop